packages/main/src/RadioButtonGroup.ts

```diff
@@ -36,7 +43,7 @@ class RadioButtonGroup {
 		const checkedRadio = this.getCheckedRadioFromGroup(groupName);
 
 		// Remove the radio button from the given group
-		group.forEach((_radioBtn, idx, arr) => {
+		group!.forEach((_radioBtn, idx, arr) => {
```

Comment:
since you are checking if the group exist at the top of the function you can move ```!``` where variable group is defined. With this typescript will stop complaining about possibly null value and you don't have to use ! more than one time.